KaijuKiller/Assets/Scripts/PlayerAnim.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerAnim : MonoBehaviour
```

Comment:
Don't think this is needed if animator is being used by the PlayerMove.cs. Or you can keep this and use SoC to make this an animator class for the player.